src/brain/issueLabelHandler/followups.ts

```diff
@@ -158,5 +166,23 @@ export async function ensureOneWaitingForLabel({
 
   await modifyProjectIssueField(itemId, labelName, STATUS_FIELD_ID, octokit);
 
+  let timeToTriageBy
+  if (labelName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
```

Comment:
move the calculation of due dates to the followups handler that listens for `Waiting for: *` labels and take it out of routing handler